hbmqtt/client.py

```diff
@@ -498,7 +497,7 @@ def _initsession(
             broker_conf['cleansession'] = cleansession
 
         for key in ['uri']:
```

Comment:
Wouldn't it be cleaner to drop this iteration over one item and replace it with:
```
            if not broker_conf.get('uri'):
                raise ClientException("Missing connection parameter 'uri'")
```

Reply:
this code indeed looks odd.

Will clean up that as well

Reply:
`broker_conf['uri']` is not even use for the connection :sweat:

Reply:
Strange. So dump it I'd say.

Reply:
Or just leave it in place to avoid bad surprises.